hstest/testing/plotting/drawing/drawing_type.py

```diff
@@ -27,10 +27,17 @@ class DrawingType:
     # https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.bar.html
     # https://seaborn.pydata.org/generated/seaborn.barplot.html
 
+    # ------
+
+    pie = "pie"
+
+    # https://matplotlib.org/stable/gallery/pie_and_polar_charts/pie_features.html
+    # https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.pie.html
+
+    # ------
```

Comment:
```suggestion
    # ------

    # https://matplotlib.org/stable/gallery/pie_and_polar_charts/pie_features.html
    # https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.plot.pie.html

    pie = "pie"

    # ------
```
Обычно комментарии пишут над кодом к которому оставлен комментарий